File: 10221 Satellites/solution.java
import java.io.*;

public class Main {
	public static void main(String[] args) throws IOException {
        StreamTokenizer st = new StreamTokenizer(System.in);
        StringBuilder output = new StringBuilder();
        while(st.nextToken() == StreamTokenizer.TT_NUMBER) {
    	    double s = st.nval;
    	    st.nextToken();
    	    double a = st.nval;
    	    st.nextToken();
    	    if(st.sval.charAt(0) == 'm')
    	        a /= 60;
    	    if(a > 180)
    	        a = 360 - a;
            output.append(String.format("%.6f %.6f\n", (6440 + s) * Math.PI * a / 180, Math.sqrt(2 * Math.pow(6440 + s, 2) * (1 - Math.cos(a * Math.PI / 180)))));
    	}
    	System.out.print(output);
	}
}

File: 10221 Satellites/solution.c
#include <stdio.h>
#include <string.h>
#include <math.h>

int main() {
    double PI = acos(-1), s, a;
    char str[4];
    while(scanf("%lf %lf %s", &s, &a, str) == 3) {
        if(str[0] == 'm')
            a /= 60;
        if(a > 180)
            a = 360 - a;
        printf("%.6f %.6f\n", (6440 + s) * PI * a / 180, sqrt(2 * pow(6440 + s, 2) * (1 - cos(a * PI / 180))));
    }
    return 0;
}
